demo/scripts/startFPCAuctionNetwork.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+#
+# Copyright IBM Corp All Rights Reserved
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+export DEMO_SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+export FPC_ROOT=${DEMO_SCRIPTS_DIR}/../..
+export DEMO_ROOT=${DEMO_SCRIPTS_DIR}/..
+
+# SCRIPT_DIR is the docker compose script dir that needs to be defined to source environment variables from the FPC Network
+export SCRIPT_DIR=${FPC_ROOT}/utils/docker-compose/scripts
+. ${SCRIPT_DIR}/lib/common.sh
+
+# Cleanup any previous iterations of the demo
+"${SCRIPT_DIR}/teardown.sh" --clean-slate
+
+# Generate the necessary credentials and start the FPC network
+"${SCRIPT_DIR}/generate.sh"
+"${SCRIPT_DIR}/start.sh"
+
+# Install and Instantiate Auction Chaincode
```

Comment:
One way we can achieve not rebuilding this is by not cleaning up the docker images in the teardown script. So once the image is built it shouldn't need to rebuild since there will be a local image. I can make that one of those `--clean-slate` flag options?

Reply:
I believe this is resolved in the sense that the teardown will not bring down the chaincode image unless `--clean-state` is given so the chaincode image does not need to be recreated every time. But as you pointed out, it is slightly unavoidable to restart from scratch if you want to re-install the CC

Reply:
you do though still a hardcoded --clean-state in the start, so the option to not requiring it in teardown is not thaaat useful?